mlir/include/mlir-c/Dialect/LLVM.h

```diff
@@ -316,7 +316,8 @@ MLIR_CAPI_EXPORTED MlirAttribute mlirLLVMDISubprogramAttrGet(
     MlirContext ctx, MlirAttribute id, MlirAttribute compileUnit,
     MlirAttribute scope, MlirAttribute name, MlirAttribute linkageName,
     MlirAttribute file, unsigned int line, unsigned int scopeLine,
-    uint64_t subprogramFlags, MlirAttribute type);
+    uint64_t subprogramFlags, MlirAttribute type, intptr_t nNodes,
+    MlirAttribute const *nodes);
```

Comment:
```suggestion
    uint64_t subprogramFlags, MlirAttribute type, intptr_t nRetainedNodes,
    MlirAttribute const *retainedNodes);
```
nit: The name should probably match the attribute fields?

Reply:
Done.